src/components/elements/heading/heading.styles.ts

```diff
@@ -4,13 +4,22 @@ import type { HeadingProps } from './heading.types';
 
 import { styled } from '@/styles';
 
+const lineHeightMap = {
+  xl: '110%',
+  l: '110%',
+  m: '110%',
+  s: '130%',
+  xs: '130%',
+};
```

Comment:
Can we move it to the theme object `src/styles/index.ts` and get the values from there, like `fontSizes` color, etc?

Reply:
Don't think would be useful since this is a map for heading specific styles and don't think it would be useful to move these values (110%, 130%) to theme config